babel.config.js

```diff
@@ -1,4 +1,4 @@
-const optimizeForSpeed = process.env.NODE_ENV === 'production'
+const optimizeForSpeed = true //process.env.NODE_ENV === 'production'
```

Comment:
Unfortunately this is necessary, otherwise styles from `ui-core` won't be rendered.
It's a bit complicated, has sth to do with styled-jsx being a singleton and `ui-core` being build in production mode and `ui-widgets` being used in dev mode